### PR TITLE
I2 borrow item control

### PR DESCRIPTION
**Pull request by amrmosit**
Hey guys,
BorrowItemControl class has been refactored to conform with CSG and class diagram. It is ready for review. 
@haynescrossley 
@elliotmitchum

### CODE REVIEW

**Review by elliotmitchum — APPROVED** [review]
Looks good to me, @haynescrossley ready to merge.

File: src/library/borrowitem/BorrowItemControl.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import library.entities.Loan;
+import library.entities.Patron;
+
+public class BorrowItemControl {
+    
+    private BorrowItemUI ui;
+    
+    private Library library;
+    private Patron patron;
+    private enum CONTROL_STATE { INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED }
```
Comment (elliotmitchum) [inline src/library/borrowitem/BorrowItemControl.java:16]: @amrmosit I missed this, CSG 4 states this should be pascal case.

File: src/library/borrowitem/BorrowItemControl.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    private CONTROL_STATE state;
```
Comment (elliotmitchum) [inline src/library/borrowitem/BorrowItemControl.java:17]: @amrmosit I missed this, CSG 4 states this should be pascal case.

Comment (amrmosit) [reply src/library/borrowitem/BorrowItemControl.java:17]: Ok. I will do this modification at the end of iteration 2 with any other required modifications. Updated Iteration2 plan

**Pull request merged**